values.md

```diff
@@ -32,7 +32,7 @@ While we believe that we are building something exceptional, that knowledge does
 
 ### We optimize for the long-term.
 
-While we pursue the mission with intense focus, we recognize that creating the change in the world that we wish to see will not happen overnight. We think, plan, and act for the long-term.
+We recognize that creating the change in the world that we wish to see will not happen overnight. Earning the right to pursue this work over decades requires both long-term vision and near-term execution, both user love and commercial success.
```

Comment:
I love what @davidharting is saying ^ about values conversing with one another and "user love and commercial success" being one example of the many tensions we need to balance. Perhaps this dichotomized example can be be represented differently, which might help us better understand how we can bring our dbt values to life in our everyday work, and help us align as a company. 

To that end, I suggest we call out the hard work of prioritizing today that will help us execute the vision for tomorrow, and making it more obvious that one is not chosen over the other, but they are dependent on each other.

```suggestion
We recognize that creating the change in the world that we wish to see will not happen overnight. Earning the right to pursue this work over decades requires balancing the tension between long-term vision and near-term execution, which includes captivating user love while pursuing commercial success.
```

User love  can't exist without commercial success (because dbt won't exist), and commercial success can't exist without user love (because we need users to love us to be successful).